internal/api/handlers/health_test.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHealthCheckHandler(t *testing.T) {
+	t.Run("should return 200 status code", func(t *testing.T) {
+		t.Parallel()
+		req, _ := http.NewRequest("GET", "/api/v1/health", nil)
```

Comment:
I know this can look a bit tedious, but asserting all returned fields in unit tests is actually very helpful, as it can often catch some wrong assumptions.